apps/portal/controllers/apis/user.jag

```diff
@@ -8,9 +8,22 @@
 
     var matcher = new URIMatcher(request.getRequestURI());
     if (matcher.match('/user')) {
+        var carbon = require('carbon');
+        var server = new carbon.server.Server();
+        var tenantId = carbon.server.tenantId();
+        var userManager = new carbon.user.UserManager(server, tenantId);
+        var adminRole = userManager.getAdminRoleName();
+        user.isAdmin = false;
+
+        for(var i = 0; i < user.roles.length; i++){
+            if(user.roles[i]== adminRole){
```

Comment:
for loops statement also should be formatted with spaces before and after brackets